episodes/02-setup.md

```diff
@@ -117,15 +177,153 @@ If you want to save your changes and quit, press <kbd>Esc</kbd> then type `:wq`
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+-->
+
+## Set up your GitHub token
+
+To interact with GitHub we need to include credentials in the request. We are going to configure one type of credential called _Personal Access Token (PAT)_. We need this to prove that we are a specific GitHub user, allowed to do whatever we’re asking to do. ([Bryan, 2021](https://happygitwithr.com/https-pat.html))
+
+__First, let's create your token.__
+
+Do this with `usethis::create_github_token()`. This function should redirect you to GitHub on your browser. Once there, check all the options in the figure below.
+
+```r
+usethis::create_github_token()
+```
+
+Describe the token use case.
+
+Check all of the following options:
+
+![](fig/git-token.png)
```

Comment:
It might be worth explaining some of the details of the options they are selecting. Nothing too detailed, but just to give a very short overview of what this PAT will allow them to do.

Reply:
I agree. I added text in https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/4ad447615fd9a70658f12f29d366a05c0c7699ee and a callout https://github.com/epiverse-trace/git-rstudio-basics/pull/13/commits/0cb5ebe621fdc23fa16d45f0d19e920da48084ab